third_party/maya/lib/usdMaya/JobArgs.h

```diff
@@ -168,12 +171,30 @@ struct JobExportArgs
     PXRUSDMAYA_API
     static const VtDictionary& GetDefaultDictionary();
 
+    PXRUSDMAYA_API
+    void addFilteredTypeName(const MString& typeName);
+
+    const std::set<unsigned int>& getFilteredTypeIds() const {
+        return filteredTypeIds;
+    }
+
+    void clearFilteredTypeIds() {
+        filteredTypeIds.clear();
+    }
+
 private:
     PXRUSDMAYA_API
     JobExportArgs(
         const VtDictionary& userArgs,
         const PxrUsdMayaUtil::ShapeSet& dagPaths,
         const GfInterval& timeInterval);
+
+    // Maya type ids to avoid exporting; these are
+    // EXACT types, though the only exposed way to modify this,
+    // addFilteredTypeName, will also add all inherited types
```

Comment:
Need to fix up this reference to addFilteredTypeName to AddFilteredTypeName per above